apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/testing/SingleTestRunPage/TrendChart.jsx

```diff
@@ -38,7 +39,12 @@ function TrendChart(props) {
             (<DateRangePicker ranges={values.ranges}
                 initialDispatch={currDateRange}
                 dispatch={(dateObj) => getDate(dateObj)}
-                setPopoverState={() => { }}
+                setPopoverState={() => {
+                    setHideFilter(true)
+                    setTimeout(() => {
```

Comment:
just remove setPopoverState this caller

Reply:
It won't work if I remove setPopoverState